### ISSUE TITLE
Mod Requests

### ISSUE THREAD

**Issue by TheOnlyTME** [DNSTechpack/DNS10-MC1.10.2#1]
I would appreciate these mods  being included.

Better Combat Mod
Better FPS (Optional)
Ender Utilities
EZ Storage 2
Inventory Pets

**Comment by robaimes** [comment]
Thanks for the requests! Next time though, if you could make an issue for each mod that would be awesome. It just helps everyone else look to see if their mod has already been requested. Should have said that in the README, my bad. I've updated the README now with a bit more info.

I do not want to take away or modify vanilla mechanics. Because of this the Better Combat Mod I feel would not fit very well into the DNS Techpack. It adds RNG in the form of critical hits plus more, which would not help those who want to play casually. 

Inventory pets I feel is superseded by the Baubles API and inventory system we already include. Baubles are not required to be on your hotbar and you can still have multiple effects.

EZ Storage is extremely similar to Integrated Dynamics, Correlated Potentialistics and Refined Storage. Because of this, I will add it to the consideration list alongside the other 3.

Better FPS is not something I want to package with DNS as it has the potential to make things unstable, especially rendering. Due to the render changes, crashes can occur when normally they shouldn't have. This is the same reason why Optifine is not included as an option.

Ender Utilities is similar to a couple of mods we have already. Random things is the main one. Actually additions is also on the consideration list. I will add this mod alongside, as there may be an argument just this or for both mods to be included.

I'll leave this open until I sort out all of the requests. Don't think I'm being harsh, I just have personal criteria I like to follow.

TL;DR:
BCM - No
Better FPS - No / Unlikely
Ender Utilities - Maybe
EZ Storage - Maybe
Inventory Pets - No
